media/js/mozorg/contribute/contribute-2015.js

```diff
@@ -155,8 +155,17 @@
                 show_area(categoryarea);
             }
         } else {
+            // get and store the previous area before collapsing and hiding
+            // it's container else, we will not be able to get att it using
+            // the :visible psuedo selector.
+            var previouslySelectedArea = $('.area:visible');
+            previouslySelectedArea.find('select')
+                .prop('selectedIndex', 0)
+                .attr('required', false);
             $areascontainer.slideUp('fast', function() {
-                $('.area:visible').hide().find('select').prop('selectedIndex', 0);
+                // instead of using .hide() which is going to change in the
+                // upcoming jQuery 3 release, just remove the style attrbiute.
+                previouslySelectedArea.removeAttr('style');
```

Comment:
This works, but if you want to make it work fully in jQuery 3, I believe we would need to use a class such as `.hidden` on each area, and then remove/add it when needed. As far as I understand, even calling `slideDown` on an element which is set to `display: none;` in the page's CSS wouldn't work. 

Up to you if you want to go that far in this PR of course (much of this page would probably need attention to make it all work).